relay-general/src/store/normalize/span/tag_extraction.rs

```diff
@@ -560,4 +565,34 @@ mod tests {
         let query = r#"UPDATE "a" SET "x" = %s, "y" = %s WHERE "z" = %s"#;
         assert_eq!(sql_table_from_query(query).unwrap(), "a");
     }
+
+    #[test]
+    fn extract_sql_action() {
+        let test_cases = vec![
+            (
+                r#"SELECT "sentry_organization"."id" FROM "sentry_organization" WHERE "sentry_organization"."id" = %s"#,
+                "SELECT",
+            ),
+            (
+                r#"INSERT INTO "sentry_groupseen" ("project_id", "group_id", "user_id", "last_seen") VALUES (%s, %s, %s, %s) RETURNING "sentry_groupseen"."id"#,
+                "INSERT",
+            ),
+            (
+                r#"UPDATE sentry_release SET date_released = %s WHERE id = %s"#,
+                "UPDATE",
+            ),
+            (
+                r#"DELETE FROM "sentry_groupinbox" WHERE "sentry_groupinbox"."id" IN (%s)"#,
+                "DELETE",
+            ),
+            (r#"SET search_path TO my_schema, public"#, "SET"),
+            (r#"SAVEPOINT %s"#, "SAVEPOINT"),
+            (r#"RELEASE SAVEPOINT %s"#, "RELEASE SAVEPOINT"),
+            (r#"ROLLBACK TO SAVEPOINT %s"#, "ROLLBACK TO SAVEPOINT"),
+        ];
+
+        for (query, expected) in test_cases.into_iter() {
```

Comment:
I believe you don't need this, as the compiler handles it:
```suggestion
        for (query, expected) in test_cases {
```